app/main.py

```diff
@@ -1,3 +1,56 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int | float) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: Distance | int | float) -> Distance:
+        if isinstance(other, (int, float)):
+            self.km += other
+        else:
+            self.km += other.km
+        return self
+
+    def __mul__(self, number: int | float) -> Distance:
+        return Distance(self.km * number)
+
+    def __truediv__(self, number: int | float) -> Distance:
+        return Distance(round((self.km / number), 2))
+
+    def __lt__(self, other: Distance) -> bool:
+        if isinstance(other, (int, float)):
+            return self.km < other
+        return self.km < other.km
+
+    def __gt__(self, other: Distance) -> bool:
```

Comment:
according to ur logic other can be not only Distance but also int and float
add related annotations